services/comm/spec/models/template_spec.rb

```diff
@@ -5,8 +5,44 @@
 RSpec.describe Template, type: :model do
   include_examples 'application record concern' do
     let(:tenant) { Tenant.first }
-    let!(:subject) { create(factory_name) }
+    let!(:first_name) { 'Jim' }
+    let!(:title) { 'Mr.' }
```

Comment:
Also, as a general rule of thumb, `let` is preferred in regards to `let!`
The main difference is that `let` is lazily evaluated (evaluated when invoked), while `let!` is evaluated at execution time. This also means that if during some test setup you use `let` and you never reference that variable, then the code is never executed which can save you some precious seconds in the long run.